test/test-readability.js

```diff
@@ -63,7 +63,7 @@ describe("Test page", function() {
         pathBase: "http://fakehost/test"
       };
 
-      beforeEach(function() {
+      before(function() {
```

Comment:
Note on this: we were parsing before *each* test, while they all rely on the same result; that was unnecessary overhead, hence this change.